languages/de-DE/Buttons.multids

```diff
@@ -30,12 +30,22 @@ ExportTiddler/Caption: Exportieren
 ExportTiddler/Hint: Diesen Tiddler Exportieren
 ExportTiddlers/Caption: Mehrere Exportieren
 ExportTiddlers/Hint: Mehrere Tiddler Exportieren
+Fold/Caption: Ausblenden Textbereich
```

Comment:
If it is in all items, then it should not be part of the options at all but the accompanying label, e.g.:

**Textbereich**:
*v ausblenden*
   einblenden
   alle ausblenden
   alle einblenden